src/server/middleware/auth.ts

```diff
@@ -1,82 +1,47 @@
-import { Json, User, authenticateJWT, parseJWT } from "@thirdweb-dev/auth";
+import { parseJWT } from "@thirdweb-dev/auth";
 import {
   ThirdwebAuth,
   ThirdwebAuthUser,
   getToken as getJWT,
 } from "@thirdweb-dev/auth/fastify";
-import { GenericAuthWallet } from "@thirdweb-dev/wallets";
 import { AsyncWallet } from "@thirdweb-dev/wallets/evm/wallets/async";
-import { utils } from "ethers";
 import { FastifyInstance } from "fastify";
 import { FastifyRequest } from "fastify/types/request";
+import jsonwebtoken from "jsonwebtoken";
 import { validate as uuidValidate } from "uuid";
 import { getPermissions } from "../../db/permissions/getPermissions";
 import { createToken } from "../../db/tokens/createToken";
 import { revokeToken } from "../../db/tokens/revokeToken";
 import { WebhooksEventTypes } from "../../schema/webhooks";
+import { THIRDWEB_DASHBOARD_ISSUER, handleSiwe } from "../../utils/auth";
 import { getAccessToken } from "../../utils/cache/accessToken";
 import { getAuthWallet } from "../../utils/cache/authWallet";
 import { getConfig } from "../../utils/cache/getConfig";
 import { getWebhook } from "../../utils/cache/getWebhook";
+import { getKeypair } from "../../utils/cache/keypair";
 import { env } from "../../utils/env";
 import { logger } from "../../utils/logger";
 import { sendWebhookRequest } from "../../utils/webhook";
 import { Permission } from "../schemas/auth";
 
+const KEYPAIR_AUTH_MAX_DURATION_SECONDS = 15 * 60;
```

Comment:
should be configurable per keypair probably?

Reply:
Yeah the original intent is no longer very valuable, will remove.